client/app/components/message_header.coffee

```diff
@@ -0,0 +1,118 @@
+{div, span, table, tbody, tr, td, img, a, i} = React.DOM
+MessageUtils = require '../utils/message_utils'
+
+ContactStore   = require '../stores/contact_store'
+
+{MessageFlags} = require '../constants/app_constants'
+
+
+(Array.prototype.last = -> this[this.length - 1]) unless Array.prototype.last
```

Comment:
OOohhhh my bad :cry:… Thanks a lot! Of course, `_.last` is **exactly** what I expected…